packages/geofire/src/GeoQuery.ts

```diff
@@ -6,22 +6,21 @@ import {
 import {
   decodeGeoFireObject, geoFireGetKey } from './databaseUtils';
 import * as GeoFireTypes from './GeoFireTypes';
-import * as DatabaseTypes from '@firebase/database-types';
+import { 
+  DataSnapshot, orderByChild, DatabaseReference, query, startAt, endAt, off, child, get, Query, onChildAdded, onChildRemoved,
+  onChildChanged, onValue, Unsubscribe 
+} from 'firebase/database';
 
 export interface QueryCriteria {
   center?: Geopoint;
   radius?: number;
 }
-export type QueryStateCallback = (
-  a: DatabaseTypes.DataSnapshot | null,
-  b?: string
-) => any;
 export interface QueryState {
   active: boolean;
-  childAddedCallback: QueryStateCallback;
-  childRemovedCallback: QueryStateCallback;
-  childChangedCallback: QueryStateCallback;
-  valueCallback: QueryStateCallback;
+  childAddedCallback: Unsubscribe;
+  childRemovedCallback: Unsubscribe;
+  childChangedCallback: Unsubscribe;
+  valueCallback: Unsubscribe;
 }
```

Comment:
We should probably rename these properties to match their type:
```suggestion
  childAddedUnsubscribe: Unsubscribe;
  childRemovedUnsubscribe: Unsubscribe;
  childChangedUnsubscribe: Unsubscribe;
  valueUnsubscribe: Unsubscribe;
}
```